### ISSUE TITLE
Rounded corners for actions

### ISSUE THREAD

**Issue by IlyasNN** [jerkoch/SwipeCellKit#410]
How to make something like this? (photo in attach)
<img width="262" alt="Screenshot 2021-02-14 at 19 34 12" src="https://user-images.githubusercontent.com/50013971/107883147-31bab400-6efe-11eb-8fb3-d5c0233952a7.png">
I've tried, but right corners could not be rounded because swipeActionsView goes outside the cell border.

**Comment by hassan31** [comment]
> How to make something like this? (photo in attach)
> <img alt="Screenshot 2021-02-14 at 19 34 12" width="262" src="https://user-images.githubusercontent.com/50013971/107883147-31bab400-6efe-11eb-8fb3-d5c0233952a7.png">
> I've tried, but right corners could not be rounded because swipeActionsView goes outside the cell border.

Yes, I'm also wondering if the option to round the corners of swipeable actions.
Referrence screenshot is attached below
![Uploading Screen Shot 2021-06-03 at 7.47.55 PM.png…]()

**Comment by salmaanahmed** [comment]
@IlyasNN were you able to achieve this?

**Comment by zeeshankhan** [comment]
any solution here?

**Comment by zeeshankhan** [comment]
any solution here?
